README.md

```diff
@@ -1 +1,172 @@
+        env:
+        - name: NOTIFICATION_TOPIC_ARN
+          value: < Topic ARN >
+        - name: NOTIFICATION_REGION
+          value: < Region >
+        volumeMounts:
+        - name: vol-secret
+          mountPath: /.aws    
+```
+
+
+### AWS credentilas as Environment Variables
+
+Another way to supply the credentials is Environment Variables.
+| Name | Description |
+| -- | -- |
+| AWS_ACCESS_KEY_ID | AWS Key ID
+| AWS_SECRET_ACCESS_KEY | AWS secret key
+
+#### Deployment example
+
+``` 
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    run: notification-controller
+  name: notification-controller
+  namespace: notification-controller
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: notification-controller
+  template:
+    metadata:
+      labels:
+        run: notification-controller
+    spec:
+      serviceAccountName: notification-controller-sa
+      containers:
+      - name: notification-controller
+        image: < Link to image >
+        env:
+        - name: AWS_ACCESS_KEY_ID
+          value: < AWS Access Key ID >
+        - name: AWS_SECRET_ACCESS_KEY
+          value: < AWS Secret Access Key >
+        - name: NOTIFICATION_TOPIC_ARN
+          value: < Topic ARN >
+        - name: NOTIFICATION_REGION
+          value: < Region >
+```
+
+## Define Topic and Region
+
+These environment variables will be used to define the `SNS topic` which the messages will be sent to and the `region` of the AWS account.
+
+| Name | Description |
+| -- | -- |
+| NOTIFICATION_REGION | define the AWS region to use
+| NOTIFICATION_TOPIC_ARN | the topic arn the messages will be sent to
+
+
+## Running, building and testing the controller
+
+This controller provides a [Makefile](Makefile) to run all the usual development tasks. This file can be used by cloning
+the repository and running `make` over any of the provided targets.
+
+### Running the controller locally
+
+When testing locally, the command `make run install` can be used to deploy and run the controller. 
+If any change has been done in the code, `make manifests generate` should be executed before to generate the new resources
+and build the controller.
+
+### Build and push a new image
+
+To build the controller and push a new image to the registry, the following commands can be used: 
+
+```shell
+$ make docker-build
+$ make docker-push
+```
+
+These commands will use the default image and tag. To modify them, new values for `TAG` and `IMG` environment variables
+can be passed. For example, to override the tag:
+
+```shell
+$ TAG=my-tag make docker-build
+$ TAG=my-tag make docker-push
+```
+
+Or, in the case the image should be pushed to a different repository:
+
+```shell
+$ IMG=quay.io/user/release:my-tag make docker-build
+$ IMG=quay.io/user/release:my-tag make docker-push
+```
+
+### Running tests
+
+To test the code, simply run `make test`. This command will fetch all the required dependencies and test the code. The
```

Comment:
I'd avoid suggesting something is simple. When it doesn't work, it will make the reader extra angry :)
```suggestion
To test the code, run `make test`. This command will fetch all the required dependencies and test the code. The
```